source/openapi3/split.yaml

```diff
@@ -2275,6 +2278,7 @@ paths:
 
 
         <aside class="notice">When creating this type of contact, the initial response <code>payid_details.state</code> value will always be <code>pending</code>. After a few seconds, it'll transition to <code>active</code>. We suggest you use webhooks to be informed of this state change</aside>
+        <aside class="notice">You can test receiving payments to a receivable contact in our Sandbox environment using the <a href="#simulate-incoming-payid-payment">PayID simulation endpoint</a>.</aside>
```

Comment:
```suggestion
        <aside class="notice">You can test receiving payments to a Receivable Contact in our sandbox environment using the <a href="/#simulate-incoming-payid-payment">PayID simulation endpoint</a>.</aside>
```

Reply:
@trev should this casing (`Sandbox` -> `sandbox`) & (`receivable contact` -> `Receivable Contact`) apply across my other changes as well?

Reply:
@compactcode - yep!